config.d.ts

```diff
@@ -18,6 +18,32 @@ import type * as playwright from 'playwright';
 
 export type ToolCapability = 'core' | 'tabs' | 'pdf' | 'history' | 'wait' | 'files' | 'install' | 'testing';
 
+type LaunchOptions = Omit<
+  playwright.LaunchOptions,
+    'handleSIGHUP'
+  | 'handleSIGINT'
+  | 'handleSIGTERM'
+  | 'logger'
+>;
+
+type ClientCertificate = Omit<
```

Comment:
I'm not sure if we want to do this. In theory there are two kinds of configs:
a) config which you pass to createConnection(config)
b) config which you pass via --config

a) does support Buffers while b) does not.

We should probably split up the config types.

Reply:
done in https://github.com/microsoft/playwright-mcp/pull/574/commits/75a1f5bfa8a6713e32f9b0a6f15c45192439e824